src/main/webapp/js/navigation-loader.js

```diff
@@ -47,6 +47,11 @@ function addLoginOrLogoutLinkToNavigation() {
 
 function addNaviMenues(){
   const navigationElement = document.getElementById('navigation');
+  if (!navigationElement) {
+    console.warn('Navigation element not found!');
```

Comment:
What would the difference between console.warn and console.log be ?

Reply:
Is it just the yellow symbol?

Reply:
It will raise warning on console. 
.log will just print

Reply:
Thank you so much. Got it !